rdr_service/model/genomics.py

```diff
@@ -583,6 +584,8 @@ class GenomicGCValidationMetrics(Base):
                                    Enum(GenomicContaminationCategory),
                                    default=GenomicSubProcessResult.UNSET)
 
+    alignmentPctHg38 = Column('alignment_pct_hg38', String(255), nullable=True)
```

Comment:
Do you mind updating this to be `alignment_pct_hg38`? The rest of the attributes in this model are snake_case.